app.js

```diff
@@ -13,14 +13,27 @@ const onCreateInputField = (todo) => {
     inputElement.classList.add("text");
     inputElement.type = "text";
     inputElement.value = todo.text;
-    inputElement.setAttribute("readonly", "readonly")
+    inputElement.setAttribute("readonly", "readonly");
+
     return inputElement;
 }
+
 const handleDelete = (todo) => {
     todos = todos.filter(todoElement => todoElement.id !== todo.id);
     renderTodos();
 }
 
+const handleEdit = (inputElement, editButton) => {
+    if(editButton.innerText === 'Edit'){
```

Comment:
Checking the `innerText` of a DOM element is very risky as an elements `innerText` can change anytime depending on requirements. 

For this project, this is fine but to follow the best practice, let's not use innerText and use data attributes.
```suggestion
    if(editButton.getAttribute("data-state") === 'edit'){
```